carapace-server/src/main/java/org/carapaceproxy/core/ProxyRequestsManager.java

```diff
@@ -522,6 +532,11 @@ private Publisher<Void> serveFromCache(ProxyRequest request, ContentsCache.Conte
             headers.add(HttpHeaderNames.EXPIRES, HttpUtils.formatDateHeader(new java.util.Date(content.getExpiresTs())));
             request.setResponseHeaders(headers);
             addCustomResponseHeaders(request, request.getAction().customHeaders);
+            //If the request is http 1.0 we make sure to send without chunked
+            if (parent.getCurrentConfiguration().isHttp10BackwardCompatibilityEnabled() &&
+                    request.getRequest().version() == HttpVersion.HTTP_1_0) {
+                return request.sendResponseData(Mono.from(ByteBufFlux.fromIterable(content.getChunks())));
```

Comment:
Is it possible to create the Mono directly from content.getChuks withot using ByteBufFlux.fromIterable?

Reply:
@pv3nturi  It's not possibile. content.getChunks is a List of ByteBuf and Mono doesn't have any function to convert a List of Bytebuf